tests/show_ip_int_test.py

```diff
@@ -0,0 +1,217 @@
+import os
+import pytest
+import subprocess
+from click.testing import CliRunner
+
+import show.main as show
+from .utils import get_result_and_return_code
+
+root_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(root_path)
+scripts_path = os.path.join(modules_path, "scripts")
+
+
+setup_ip_intfs_single_asic = [
+#    # ipv6 is disabled by default ,
+    'sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0',
+    # this flag is need to stop kernel auto generating Linklocal address
+    'sudo sysctl -w net.ipv6.conf.default.addr_gen_mode=1',
+    # dummy physical port
+    'sudo ip link add Ethernet0 type dummy',
+    'sudo ip addr add 20.1.1.1/24 dev Ethernet0',
+    'sudo ip -6 address add dev Ethernet0 scope link fe80::64be:a1ff:fe85:c6c4/64',
+    'sudo ip -6 addr add aa00::1/64 dev Ethernet0',
+    'sudo ip link set Ethernet0 up',
+
+    #dummy portchannel
+    'sudo ip link add PortChannel0001 type dummy',
+    'sudo ip addr add 30.1.1.1/24 dev PortChannel0001',
+    'sudo ip -6 address add dev PortChannel0001 scope link fe80::cc8d:60ff:fe08:139f/64',
+    'sudo ip -6 addr add ab00::1/64 dev PortChannel0001',
+    'sudo ip link set PortChannel0001 up',
+
+    #dummy Vlan interface
+    'sudo ip link add Vlan100 type dummy',
+    'sudo ip addr add 40.1.1.1/24 dev Vlan100',
+    'sudo ip -6 address add dev Vlan100 scope link fe80::c029:3fff:fe41:cf56/64',
+    'sudo ip -6 addr add cc00::1/64 dev Vlan100',
+    'sudo ip link set Vlan100 up',
+
+]
+cleanup_ip_intfs_single_asic = [
+    'sudo ip link del Ethernet0',
+    'sudo ip link del PortChannel0001',
+    'sudo ip link del Vlan100',
+    'sudo sysctl -w net.ipv6.conf.default.addr_gen_mode=1',
+    'sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1'
+
```

Comment:
Remove this blank line

Reply:
looks like we have an issue with using sysctl in docker, @jleveque ?

Reply:
Yes. Good catch. Discussion is here: https://github.com/Azure/sonic-utilities/pull/1406. We need to figure out a solution.

Reply:
@jleveque , @prsunny 
I am able to use the sysctl commands in the slave docker. 
```
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$ sudo sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 1
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$ sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0
net.ipv6.conf.all.disable_ipv6 = 0
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$ sudo sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 0
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$ sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1
net.ipv6.conf.all.disable_ipv6 = 1
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$ sudo sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 1
arlakshm@a0b00d89505a:/sonic/src/sonic-utilities$
```
We do similar configuration for the [ptf docker in sonic-mgmt ](https://github.com/Azure/sonic-mgmt/blob/1e70c8ea29ad0c071f0b36f48315ab5f2a8b5d5c/ansible/roles/vm_set/tasks/add_topo.yml#L75)

Reply:
I wanted to use 'netifaces' apis in the tests instead of mocking the data.  If this might cause as issue, let me know I can rewrite the tests

Reply:
> @jleveque , @prsunny
> I am able to use the sysctl commands in the slave docker

Interesting. I wonder what the difference is compared to #1406. Maybe because Azure Pipelines doesn't run the container in privileged mode?

Reply:
@arlakshm: I have confirmed that "privileged" mode is the difference. However, please see the comments [here](https://github.com/Azure/sonic-utilities/pull/1406#issuecomment-775705782). Tests which rely on interacting with the system should be reserved for sonic-mgmt tests. For unit testing, you should simply mock any system calls and return contrived output that you can test against.

Reply:
@jleveque, update the unit tests to use mock data in the latest commit.